test/table.test.tsx

```diff
@@ -718,4 +718,90 @@ describe('printTable compatibility with @oclif/test', () => {
     )
     expect(stdout).to.equal(expected)
   })
+
+  it('prints full table with no set width', async () => {
+    console.log(process.stdout.columns)
+    const data = [
+      {name: 'Foo', age: '1'.repeat(100)},
+      {name: 'Bar', age: '2'.repeat(100)},
+    ]
+
+    const expected = `┌──────┬──────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ name │ age                                                                                                  │
+├──────┼──────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│ Foo  │ 1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111 │
+├──────┼──────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│ Bar  │ 2222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222 │
+└──────┴──────────────────────────────────────────────────────────────────────────────────────────────────────┘
+
+`
+
+    const {stdout} = await captureOutput(async () =>
+      printTable({
+        data,
+        columns: ['name', 'age'],
+      }),
+    )
+    expect(stdout).to.equal(expected)
```

Comment:
this is the UT failing in GHA, it's not idempotent (the expected output string is static)